app/code/core/Mage/Customer/Block/Account/Dashboard/Newsletter.php

```diff
@@ -34,9 +34,17 @@
 
 class Mage_Customer_Block_Account_Dashboard_Newsletter extends Mage_Core_Block_Template
 {
+    /**
+     * @var Mage_Newsletter_Model_Subscriber
+     */
+    public $_subscription;
```

Comment:
Variables with underscore are hint for private attributes.

See [PEAR Manual](https://pear.php.net/manual/en/standards.naming.php)

Reply:
This var wasnt set and public before. Make it protected/private could cause problem with code relying on this?

Reply:
Fixed in #701